GeorgianMemenator/Scenes/MainView/MainView.swift

```diff
@@ -64,6 +64,13 @@ struct MainView: View {
                         ) {
                             CategoryRow(category: viewModel.categories[4])
                         }
+                        
+                        NavigationLink(destination:
+                                        CatTagsListView()
```

Comment:
Whenever you will add the viewModel property in `CatTagsListView`, you will need to pass instance of viewmodel here.